app/src/main/java/org/sopt/dosopttemplate/presentation/home/HomeFragment.kt

```diff
@@ -33,26 +33,33 @@ class HomeFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        initAdapter()
+        initUserData()
     }
 
-    private fun initAdapter() {
-        myProfileAdapter = MyProfileAdapter(requireContext()).apply {
-            setMyProfile(viewModel.profileData)
-        }
+    private fun initUserData() {
+        CoroutineScope(Dispatchers.IO).launch {
```

Comment:
```suggestion
        lifecycleOwner.lifecycleScope.launch {
```
오 코루틴을 사용하셨군요!! 그냥 스레드를 지정해서 코루틴 스코프를 생성하는 것보다
생명주기를 인식하는 `lifecycleOwner.lifecycleScope`를 사용하는 것을 추천드릴 것 같습니다.

아래 참고링크 첨부해드려요

https://developer.android.com/topic/libraries/architecture/coroutines?hl=ko

Reply:
네 !!!!!!!!!! 이거 나에게 필요햇던 것 ... 감사합니다